fastly/resource_fastly_service_compute_test.go

```diff
@@ -104,6 +104,10 @@ func TestAccFastlyServiceCompute_basic(t *testing.T) {
 					testAccCheckServiceV1Exists("fastly_service_compute.foo", &service),
 					resource.TestCheckResourceAttr(
 						"fastly_service_compute.foo", "name", name),
+					resource.TestCheckResourceAttr(
+						"fastly_service_compute.foo", "default_ttl", "0"),
+					resource.TestCheckResourceAttr(
+						"fastly_service_compute.foo", "stale_if_error", "true"),
```

Comment:
I presume we also want to validate that `stale_if_error_ttl` is set to zero by the configuration (as the default value of `43200 ` is being validated in the following new test).

Reply:
Oh I see you have another test for that (`TestAccFastlyServiceCompute_createZeroDefaultTTL`) but maybe worth still adding a check for the attribute here too?